test/WebJobs.Script.Tests/TestScripts/Node/BlobTriggerToBlob/function.json

```diff
@@ -0,0 +1,16 @@
+﻿{
+  "bindings": [
+    {
+      "type": "blobTrigger",
+      "direction": "in",
+      "dataType": "binary",
```

Comment:
This is the new capability - I declare that I expect binary data, so my Node script gets a Buffer, whereas before we were always converting to string, which doesn't work for things like images, etc.